------------------------------------------------------------
revno: 14147
revision-id: squid3@treenet.co.nz-20170226085617-89jfjndt62i83qtn
parent: squid3@treenet.co.nz-20170226085245-vcvc7nemupizbe4t
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Sun 2017-02-26 21:56:17 +1300
message:
  ext_kerberos_ldap_group_acl: fix unused value warnings
  
   Detected by Coverity Scan. Issues 1364748 and 1364749.
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170226085617-89jfjndt62i83qtn
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: fd770175ba851fc62de4a085414c8fd996b53e46
# timestamp: 2017-02-26 09:51:04 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20170226085245-\
#   vcvc7nemupizbe4t
# 
# Begin patch
=== modified file 'helpers/external_acl/kerberos_ldap_group/support_ldap.cc'
--- helpers/external_acl/kerberos_ldap_group/support_ldap.cc	2017-01-01 00:16:45 +0000
+++ helpers/external_acl/kerberos_ldap_group/support_ldap.cc	2017-02-26 08:56:17 +0000
@@ -919,8 +919,8 @@
     /*
      * Initialise ldap
      */
-    ldap_debug = 127 /* LDAP_DEBUG_TRACE */ ;
-    ldap_debug = -1 /* LDAP_DEBUG_ANY */ ;
+//    ldap_debug = 127 /* LDAP_DEBUG_TRACE */ ;
+//    ldap_debug = -1 /* LDAP_DEBUG_ANY */ ;
     ldap_debug = 0;
     (void) ldap_set_option(NULL, LDAP_OPT_DEBUG_LEVEL, &ldap_debug);
 #endif

